tfx/proto/example_gen.proto

```diff
@@ -71,6 +71,9 @@ enum PayloadFormat {
 
   // Serialized tf.train.Example protocol buffer.
   FORMAT_TF_EXAMPLE = 6;
+  
+  // Serialized tf.train.SequenceExample protocol buffer.
+  FORMAT_TF_SEQUENCE_EXAMPLE = 7;
```

Comment:
7 is reserved below. Please use 12 and update line 81

Reply:
got it, i made the change

Reply:
I think internally we use 7, do they need to align? 
https://screenshot.googleplex.com/Vqc7OWfpefa

Reply:
to align with internal, i removed the reserved 7 below